projects/detr3d/configs/detr3d_res101_gridmask_dev-1.x_test_old.py

```diff
@@ -0,0 +1,283 @@
+_base_ = [
+    # '.../mmdetection3d/configs/_base_/datasets/nus-3d.py',
+    '../../../configs/_base_/default_runtime.py'
+]
+
+custom_imports = dict(imports=['projects.detr3d.mmdet3d_plugin'])
```

Comment:
This cfg file can inherent from `configs/detr3d_res101_gridmask_dev-1.x.py` if there are a few modifications.

Reply:
Fixed.